java/compaction/compaction-job-execution/src/main/java/sleeper/compaction/job/execution/JavaCompactionRunner.java

```diff
@@ -130,7 +130,7 @@ private List<CloseableIterator<Row>> createInputIterators(CompactionJob compacti
             inputIterators.add(rowIterator);
             LOGGER.debug("Compaction job {}: Created reader for file {}", compactionJob.getId(), file);
             LOGGER.debug("Compaction job {}: File is being filtered on ranges {}", compactionJob.getId(),
-                    partition.getRegion().getRanges().toString());
+                    region.getRanges().toString());
```

Comment:
This is passing the result of toString to a logging call. We don't need to call toString, and if we don't the ranges will only be converted to a string when debug logs are enabled. That avoids the extra work of converting to a string if debug logs are not needed.